apps/gbt-regression/README.rst

```diff
@@ -0,0 +1,30 @@
+gbt-regression
+==============
+
+This app creates and fits an `XGBoost Gradient Boosted Tree`_ model
```

Comment:
The formatting on this seems a bit off (e.g. see https://github.com/juntai-zheng/mlflow-apps/blob/beff7f4bc759a5aca0a1923dd336efe975732844/apps/gbt-regression/README.rst), I think in RST bulleted lists are done with dashes